pkg/util/mount/mount.go

```diff
@@ -76,3 +80,40 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	}
 	return refs, nil
 }
+
+// given a device path, find its reference count from /proc/mounts
+func GetDeviceRefCount(mounter Interface, deviceName string) (int, error) {
```

Comment:
no, it's just general feedback - public functions should have godoc-compatible comments